common/src/main/scala/Configuration.scala

```diff
@@ -220,4 +223,109 @@ case class Configuration(
       employeePaid
     )
 
+      def calculateUnofficialDeferment(
+                                        taxYear: Int,
+                                        config: TaxYearBandLimits,
+                                        rows: List[UnofficialDefermentRowInput],
+                                        userDefinedBandLimits: List[Class1BandLimit]
+                                      ) =
+        UnofficialDefermentResult(
+          taxYear,
+          config,
+          rows,
+          userDefinedBandLimits
+        )
```

Comment:
I think we may need some unit tests around this before we go moving the hard coded values into configuration. 
That could be under a separate PR though, if you prefer. Let me know.

Reply:
it might be worth doing unit tests in another PR - it might take a while to get some test cases together